index.css

```diff
@@ -0,0 +1,225 @@
+        height: 95%; 
+        align-items: center;   
+      }
+
+      .aside-bottom > div {
+        padding: 10px;
+      }
+
+  
+      .a {
+          text-align: center;
+          background-color: #ffcc5c;
+          grid-column: col / span 2;
+      }
+  
+      .b {
+          background-color: #8dcfb1;
+          grid-column: col 3 / -1;
+      }
+  
+      .c {
+          text-align: center;
+          background-color: #ff6f69;
+          grid-column: col / span 2;
+          grid-row: 2 / 6;
+      }
+  
+      .d{
+          background-color: rgb(255, 182, 86);
+          grid-column: col 3 / -1;
+          grid-row: 2 / 6;
+          display: grid;
+          grid-gap: 10px;
+          padding: 20px;
+          justify-content: center;
+          align-content: center;
+          grid-template-columns: 1fr 3fr 2fr;
+          grid-template-rows: 1fr 1fr 1fr 1fr;
+      }
+
+      .d > div {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+  
+      .e {
+          background-color: #e67b90;
+          grid-column: 1 / 1;
+          grid-row: 1 / 3;
+      }
+  
+      .f {
+          background-color: #ae7ece;
+          grid-column: 2 / 2;
+          grid-row: 1 / 3;
+      }
+  
+      .g {
+          background-color: #9dcf90;
+          grid-column: 3 / 3;
+          grid-row: 1 / 5;
+      }
+
+      .h {
+        background-color: #8fdbdb;
+          grid-column: 1 / 3;
+          grid-row: 3 / 5; 
+      }
+
+      .j {
+          text-align: center;
+        background-color: #a9e069;
+        grid-column: col 1 / -1;
+        grid-row: 6 / 7;
+      }
+
+
+      @media all and (max-width: 1192px) {
+        .d{
+          background-color: rgb(255, 182, 86);
+          grid-column: col 3 / -1;
+          grid-row: 2 / 6;
+          display: grid;
+          grid-gap: 10px;
+          padding: 20px;
+          justify-content: center;
+          align-content: center;
+          grid-template-columns: 2fr 2fr 2fr;
+          grid-template-rows: 1fr 1fr 1fr 1fr;
+        }
+        
+        .e {
+          background-color: #e67b90;
+          grid-column: 1 / 2;
+          grid-row: 1 / 3;
+        }
+  
+        .f {
+            background-color: #ae7ece;
+            grid-column: 2 / 4;
+            grid-row: 1 / 3;
+        }
+  
+        .g {
+            background-color: #9dcf90;
+            grid-column: 1 / 3;
+            grid-row: 3 / 5;
+        }
+
+        .h {
+          background-color: #8fdbdb;
+            grid-column: 3 / 4;
+            grid-row: 3 / 5; 
+        }
+
+      }
+
+      @media all and (max-width: 615px) {
+        .header {
+          flex-wrap: wrap;
+        }
+
+        .header > li {
+          flex: 1 1 50%;
+          text-align: center;
+        }
+        
+        .galerry > li {
+          text-align: center;
+        }
+      }
+
+      @media all and (max-width: 652px) {
+        .e {
+          background-color: #e67b90;
```

Comment:
Why do you set up same background color? It was already specified above. Do not repeat yourself